### DISCUSSION TITLE
SQLCipher

### DISCUSSION THREAD

**Discussion by DominicPicard** [fnc12/sqlite_orm#1118] [Q&A]
Hi,
I want to use sqlite_orm for a C++ project and I was wondering if the it supports opening a connection to a encrypted database with SQLCipher ?

Thank you

**Comment by fnc12** [comment]
yes. You have to override `storage.on_open` value

**Reply by DominicPicard** [reply]
Thank you for your quick reply, it works perfectly 😃
